projection_node:
  ros__parameters:
    plane_n: [0.0, 0.0, 1.0]
    plane_d: 0.0
    pixels_per_unit: 500.0
    depth_priority_far: false
    origin_mode: "mean"
    output_file: "/home/jack/ros2_ws/output"
    input_file: "/home/jack/ros2_ws/project_hj_v2/241108_converted - Cloud.ply"
